disttask/framework/storage/task_table.go

```diff
@@ -352,10 +352,31 @@ func (stm *TaskManager) UpdateErrorToSubtask(tidbID string, taskID int64, err er
 	return err1
 }
 
-// PrintSubtaskInfo log the subtask info by taskKey.
-func (stm *TaskManager) PrintSubtaskInfo(taskKey int64) {
+// PrintSubtaskInfo log the subtask info by taskKey. Only used for UT.
+func (stm *TaskManager) PrintSubtaskInfo(taskID int64) {
 	rs, _ := stm.executeSQLWithNewSession(stm.ctx,
-		"select * from mysql.tidb_background_subtask where task_key = %?", taskKey)
+		"select * from mysql.tidb_background_subtask where task_key = %?", taskID)
+
+	for _, r := range rs {
+		errBytes := r.GetBytes(13)
+		var err error
+		if len(errBytes) > 0 {
+			stdErr := errors.Normalize("")
+			err1 := stdErr.UnmarshalJSON(errBytes)
+			if err1 != nil {
+				err = err1
+			} else {
+				err = stdErr
+			}
+		}
+		logutil.BgLogger().Info(fmt.Sprintf("subTask: %v\n", row2SubTask(r)), zap.Error(err))
+	}
+}
+
+// PrintHistorySubtaskInfo log the history subtask info by taskKey. Only used for UT.
+func (stm *TaskManager) PrintHistorySubtaskInfo(_ int64) {
```

Comment:
Could we use a unified function for `PrintSubtaskInfo` and `PrintHistorySubtaskInfo`?